integration/hello_command_test.go

```diff
@@ -0,0 +1,203 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integration
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+
+	"github.com/FerretDB/FerretDB/integration/setup"
+	"github.com/FerretDB/FerretDB/integration/shareddata"
+	"github.com/FerretDB/FerretDB/internal/types"
+	"github.com/FerretDB/FerretDB/internal/util/must"
+)
+
+func TestHello(t *testing.T) {
+	t.Parallel()
+
+	ctx, collection := setup.Setup(t, shareddata.Scalars, shareddata.Composites)
+	db := collection.Database()
+
+	var res bson.D
+
+	require.NoError(t, db.RunCommand(ctx, bson.D{
+		{"hello", "1"},
+	}).Decode(&res))
+
+	actual := ConvertDocument(t, res)
+
+	keys := []string{
+		"isWritablePrimary",
+		"maxBsonObjectSize",
+		"maxMessageSizeBytes",
+		"maxWriteBatchSize",
+		"localTime",
+		"connectionId",
+		"minWireVersion",
+		"maxWireVersion",
+		"readOnly",
+		"ok",
+	}
+
+	if !setup.IsMongoDB(t) {
+		assert.Equal(t, actual.Keys(), keys)
+	} else {
+		// MongoDB have additional keys,
+		// so just check if the keys exists on it.
+		for _, wantKey := range keys {
+			assert.Contains(t, actual.Keys(), wantKey)
+		}
+	}
+}
```

Comment:
I think that having two existing styles of checking response fields is already enough; no need to add another one. We also need to check values, not just keys.

One options is to do the same as `TestCommandsReplication`.
Another – as `TestCommandsAdministrationServerStatus`.